### DISCUSSION TITLE
[WASM] Disabling Uno0001 warnings in XAML?

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#9593] [Q&A] [answered]
I do have a large XAML file which does all the Material styling. Unfortunately I get gazziliions of warnings like

```3>C:\Users\dierk\Documents\Unusual\Unusual.Wasm\Uno.UI.SourceGenerators\Uno.UI.SourceGenerators.XamlGenerator.XamlCodeGenerator\MaterialTheme_0633246ce0a72bf767186235048cb91f.cs(1175,44,1175,111): warning Uno0001: Microsoft.UI.Xaml.Controls.Control.CharacterSpacingProperty is not implemented in Uno```

Is there a way I could disable those warnings in/for that particular XAML file? Believe '#pragma warning' is not working in XAML, no?

**Comment by jeromelaban** [comment]
Generate files are not supposed to be raising warnings, and it may be related to a project specific configuration. You can adjust the warnings in generated code using [this configuration](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/troubleshooting-source-generation.md#adding-generated-files-c-pragma), though.

**Reply by DierkDroth** [reply]
@jeromelaban hmm ... I added section below to my WASM .csproj but still am getting the same warnings.

What am I doing wrong?

```
	<ItemGroup>
		<XamlGeneratorAnalyzerSuppressions Include="Uno0001" />
	</ItemGroup>	
```

**Reply by jeromelaban** [reply]
This was updated in Uno 4.5 dev builds, if it does not otherwise work, there's no workaround available yet.

**Reply by DierkDroth** [reply]
@jeromelaban I created a repo (below) using latest Uno Dev 4.5 bits to demonstrate the problem. 
I even added 'XamlGeneratorAnalyzerSuppressions' to the project file to no avail.

[Uno0001.zip](https://github.com/unoplatform/uno/files/9432884/Uno0001.zip)

**Reply by jeromelaban** [reply]
Let's track in https://github.com/unoplatform/uno/issues/9627

**Reply by DierkDroth** [reply]
Thanks @jeromelaban

**Comment by DierkDroth** [comment, marked as answer]
Closed as per [here](https://github.com/unoplatform/uno/issues/9627)
